### DISCUSSION TITLE
Is it possible to replace laravel default config files using this package?

### DISCUSSION THREAD

**Discussion by Xinecraft** [spatie/laravel-settings#22] [Q&A] [answered]
Hi there,
             In my app, I need to let superadmin change some laravel config like SMTP mail settings, storage driver etc. so I need to override them from settings.

Is there any way to handle those using this package?

Thanks.

**Comment by rubenvanassche** [comment, marked as answer]
I think it should be possible, if you add a service provider that you'll add to the `app.php` config file where you resolve the settings class. You can then in this service provider set the SMTP configuration using Laravel's `config` helper with the values of your settings class.
